cmd/internal/shared/config.go

```diff
@@ -76,9 +76,12 @@ var DefaultGRPCConfig = config.GRPC{
 
 // DefaultRedisConfig is the default config for Redis.
 var DefaultRedisConfig = config.Redis{
-	Address:   "localhost:6379",
-	Database:  0,
-	Namespace: []string{"ttn", "v3"},
+	Address:            "localhost:6379",
+	Database:           0,
+	Namespace:          []string{"ttn", "v3"},
+	EnableFailover:     false,
+	FailoverMasterName: "mymaster",
```

Comment:
I think it's cleaner if we require the user to specify something rather than `mymaster`. Our aim is to keep default settings sane for local use.

Reply:
I had added `mymaster` as default because it is the typical default master name for Redis Sentinel. I agree with your suggestion, removed the default value